lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2375,6 +2375,9 @@ def bar3d(self, x, y, z, dx, dy, dz, color=None,
         shade : bool, optional (default = True)
             When true, this shades the dark sides of the bars (relative
             to the plot's source of light).
+            
+        lightsource : matplotlib.color.LightSource, optional (default = None)
+            The lightsource defines the plot's source of light.
```

Comment:
The lightsource defines the plot's source of light, if *shade=True* is set.

Reply:
Thanks, I noticed that `plot_trisurf` already uses this parameter so I copied its line instead:
```
lightsource : `~matplotlib.colors.LightSource`
The lightsource to use when *shade* is True.
```